src/instances/types/Component/Instance.ts

```diff
@@ -9,6 +9,14 @@ import reconcile, { shouldUpdate } from './reconcile';
 // if a file doesn't export anything other than types, it won't generate the .d.ts file
 nothing;
 
+/**
+ * ComponentInstances are used representing the <Component /> in the shadowdom
+ * or when plusnew.createElement gets called with a function
+ * it calls the constructure function and keeps the informations what dependencies the component has
+ * the render-function of the component gets called immediately after the render function
```

Comment:
should it be: render-function gets called immediately after the constructor-function

Reply:
thanks good find, fixed it